### PR TITLE
Metric adjustments

### PR DESCRIPTION
**Pull request by gestchild**
- Added Explore Landing page
- Removed old Blog page
- Changed location of test to Manchester
- Changed frequency of test from twice to once a day

### CODE REVIEW

File: _profiles/v1-article.html [hunk 1]
```diff
@@ -9,21 +9,21 @@
...
 
 #
 # WebPageTest test parameters
 #
 parameters:
   connectivity: "Cable"
-  location: "Dulles:Chrome"
+  location: "ec2-eu-west-1"
   runs: 1
   url: "https://wellcomecollection.org/articles/evolution-and-influence-adamson-collection"
```
Comment (jamesgorrie) [inline _profiles/v1-article.html]: Could we not remove this altogether as we're not using it for any reporting, or to make any decisions?

File: _profiles/v2-article.html [hunk 2]
```diff
@@ -38,6 +38,10 @@
 #
 budgets:
   -
+    metric: TTFB
```
Comment (jamesgorrie) [inline _profiles/v2-article.html:41]: The reason I didn't go for this in the first place is it's a bit of a userless metric.
[There's a bit of a blog here about it](https://blog.cloudflare.com/ttfb-time-to-first-byte-considered-meaningles/).

File: _profiles/v2-explore.html [hunk 3]
```diff
@@ -3,29 +3,29 @@
 # The human-friendly name of the profile. Whenever you're asked for the profile
 # ID, it refers to the name of the profile file — in this case, "default".
 #
-name: "WP: Article"
+name: "V2: Explore landing"
```
Comment (jamesgorrie) [inline _profiles/v2-explore.html]: If we aren't comparing to V1 any more, is the concept of V2 redundant?

**Review by jamesgorrie — COMMENTED** [review]
A few questions rather than suggestions...

File: _profiles/v1-article.html (hunk 1 above)
Comment (gestchild) [reply _profiles/v1-article.html]: Yeah, was just keeping for posterity I guess, but we don't really need it

File: _profiles/v2-article.html (hunk 2 above)
Comment (gestchild) [reply _profiles/v2-article.html:41]: Understand that it doesn't have much value on it's own. The reason I put it in was that, looking at the webpagetest results we've been getting to date, there is a huge correlation between this value and the ones we do care about (start render and speed index).  Having this value alert gives us a quick indication of where any problems may lie.

File: _profiles/v2-explore.html (hunk 3 above)
Comment (gestchild) [reply _profiles/v2-explore.html]: yeah, I'll remove the old ones and rename

File: _profiles/v2-article.html (hunk 2 above)
Comment (jamesgorrie) [reply _profiles/v2-article.html:41]: The shouldn't we be using it for analysis rather than alerting?

Comment (gestchild) [reply _profiles/v2-article.html:41]: yep, I just thought it was a quick way of analysing (in slack) whether the alerts from the other metrics were likely due to this one, without having to go and check it out. Happy to remove it if you'd rather

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**